pipelines/rh-push-to-registry-redhat-io/rh-push-to-registry-redhat-io.yaml

```diff
@@ -113,6 +113,32 @@ spec:
           workspace: release-workspace
       runAfter:
         - verify-access-to-resources
+    - name: check-data-keys
+      params:
+        - name: dataPath
+          value: "$(tasks.collect-data.results.data)"
+        - name: schema
+          value: $(params.taskGitUrl)/raw/$(params.taskGitRevision)/schema/dataKeys.json
+        - name: systems
+          value: 
+            - pyxis
+            - mapping
+            - sign
+            - cosignSecretName
+      taskRef:
+        params:
+          - name: url
+            value: $(params.taskGitUrl)
+          - name: revision
+            value: $(params.taskGitRevision)
+          - name: pathInRepo
+            value: tasks/check-data-keys/check-data-keys.yaml
+        resolver: git
+      workspaces:
+        - name: data
+          workspace: release-workspace
+      runAfter:
+        - collect-data
```

Comment:
In rh-advisories, this has verify-enterprise-contract in runAfter. Not sure which one is more correct, but they should do the same - and if run after verify-enterprise-contract is correct, then perhaps it should be used in other pipelines that have it as well.

Reply:
They all should be run after `collect-data`  ... Will update the `push-to-cdn`
done

Reply:
My comment is about rh-advisories ;) Not sure what you mean by "will update push-to-cdn" - that's a task. Which pipeline are you talking about?

Reply:
I am confused  as ` rh-advisories ` has  populate-release-notes-images in its run after for check-data-leys.

Reply:
Yeah, and we need to keep it that way AFAIK because it checks some of the keys the populate-release-notes-images task adds

Reply:
> we need to keep it that way AFAIK because it checks some of the keys the populate-release-notes-images task adds

You mean if we checked the keys after populate-release-notes-images, it might not pass? But we just discussed elsewhere that the schema does include the releaseNotes.content.images, right? So it should be fine?

But I'm not sure who was even proposing we do the check after populate-release-notes-images here, certainly not me. 

My original question still stands: rh-advisories has verify-enterprise-contract in runAfter. Do we want that, or do we want collect-data? (Of course if we decide to have it after populate-release-notes-images in rh-advisories, this question would no longer be relevant.)

Reply:
rh-advisories does not have that. Are you looking at the right pipeline? It has this
```
    - name: check-data-keys
      params:
        - name: dataPath
          value: "$(tasks.collect-data.results.data)"
        - name: schema
          value:  $(params.taskGitUrl)/raw/$(params.taskGitRevision)/schema/dataKeys.json
        - name: systems
          value:
            - releaseNotes
      taskRef:
        params:
          - name: url
            value: $(params.taskGitUrl)
          - name: revision
            value: $(params.taskGitRevision)
          - name: pathInRepo
            value: tasks/check-data-keys/check-data-keys.yaml
        resolver: git
      workspaces:
        - name: data
          workspace: release-workspace
      runAfter:
        - populate-release-notes-images
```

Reply:
> Are you looking at the right pipeline?

Thanks. I'm not sure where I was looking. You're right it's not there. Sorry about the noise.

But I do see it in `push-disk-images-to-cdn` pipeline:
```
    - name: check-data-keys
      params:
        - name: dataPath
          value: "$(tasks.collect-data.results.data)"
        - name: systems
          value:
            - cdn
      taskRef:
        params:
          - name: url
            value: $(params.taskGitUrl)
          - name: revision
            value: $(params.taskGitRevision)
          - name: pathInRepo
            value: tasks/check-data-keys/check-data-keys.yaml
        resolver: git
      workspaces:
        - name: data
          workspace: release-workspace
      runAfter:
        - verify-enterprise-contract

```

Do we know why? Shouldn't it also run right after collect-data there?

Reply:
I would imagine it should run right after the collect-data like the rest.

Reply:
Yeah, I'd do that.

Reply:
Changed it then